### PR TITLE
README update

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,131 @@
...
+Running Locally
+---------------
+
+### Prerequisites
+
+To run Alexa locally, make sure that you have the following dependencies installed: 
+
+- [nvm](https://github.com/creationix/nvm).
+
+Run `npm use` to set the correct node version and `npm install` to install the dependencies.
```
Comment (mchv) [inline README.md]: Should be run `nvm install` to install and use the correct node version.

Comment (shtukas) [inline README.md]: Done.

**Comment by mchv** [comment]
👍 when build succeed

**Comment by shtukas** [comment]
Thanks for the review. I just reran the build and it was fine.

**Pull request merged**